docs/debugging.md

```diff
@@ -1,16 +1,26 @@
 # Debugging
 
-## Warnings
+## System checks
 
 <dl>
   <dt>`flags.E001`</dt>
   <dd>
-    <p>Django-Flags includes a <a href="https://docs.djangoproject.com/en/2.2/topics/checks/">Django system check</a> to check for flags that have non-existent conditions on start-up.</p>
-    <pre><code>?: (flags.E001) Flag FLAG_WITH_ANY_CONDITIONS has non-existent condition "condition name"
+    <p>Django-Flags includes a <a href="https://docs.djangoproject.com/en/stable/topics/checks/">Django system check</a> to check for flags that have non-existent conditions on start-up.</p>
+    <pre><code>?: (flags.E001) Flag FLAG_WITH_ANY_CONDITIONS has non-existent condition "condition name".
 HINT: Register "condition name" as a Django-Flags condition.</code></pre>
   </dd>
 </dl> 
 
+<dl>
+  <dt>`flags.E002`</dt>
+  <dd>
+    <p>Django-Flags includes a <a href="https://docs.djangoproject.com/en/stable/topics/checks/">Django system check</a> to check to ensure that flag conditions have valid expected values on start-up.</p>
+    <pre><code>?: (flags.E002) Flag FLAG_WITH_ANY_CONDITIONS's "boolean" condition has an invalid value.
+HINT: Enter one of "on", "off", "true", "false", etc.</code></pre>
```

Comment:
It might not be helpful to include the vague "etc." here, because it implies you can enter other bool values, such as 1 or 0, which `strtobool` won't accept if they aren't strings. Rather than spell out all the possible options that `strtobool` accepts (which is a long list if you include upper and lowercase variations), I would omit the "etc." and offer the most useful choices. 

Also, in the [validation code'(https://github.com/cfpb/django-flags/blob/master/flags/conditions.py#L63), we don't need `.lower()` because `strtobool` does that.

Reply:
I see the validator steps around my concern about entering integers, but my objection to 'etc.' remains.

Reply:
Fixed in 6ee9d26.